Resolves "Error 2" caused by "error: size of array assertion_failed__1131 is negative"
For more information on the bug see:
* https://bugs.gentoo.org/708430
* https://github.com/easybuilders/easybuild-easyconfigs/issues/14998
Adopted from https://bugs.gentoo.org/708430#c7 by Alexander Michels, CyberGIS Center, UIUC
--- llvm-9.0.1.src/projects/compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.cc.orig	2020-03-11 20:35:37.099808821 +0100
+++ llvm-9.0.1.src/projects/compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.cc	2020-03-11 20:36:08.279808758 +0100
@@ -1128,7 +1128,7 @@
 CHECK_SIZE_AND_OFFSET(ipc_perm, cgid);
 #if !defined(__aarch64__) || !SANITIZER_LINUX || __GLIBC_PREREQ (2, 21)
 /* On aarch64 glibc 2.20 and earlier provided incorrect mode field.  */
-CHECK_SIZE_AND_OFFSET(ipc_perm, mode);
+//CHECK_SIZE_AND_OFFSET(ipc_perm, mode);
 #endif
 
 CHECK_TYPE_SIZE(shmid_ds);
